.github/workflows/download_oas_files.yml

```diff
@@ -0,0 +1,75 @@
+name: Download OpenAPI Files and Update Config
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1" # Runs weekly at midnight on Monday
+
+jobs:
+  update_openapi_and_config:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Download OpenAPI files
+        run: |
```

Comment:
Would it be possible / better to maintain this list of json files in a config file? (Just to make it easier to find where to add a new api spec).

Could be done in the next iteration though